appinventor/components/src/com/google/appinventor/components/common/YaVersion.java

```diff
@@ -411,6 +411,8 @@ private YaVersion() {
   // - FTC_DC_MOTOR_CONTROLLER_COMPONENT_VERSION was incremented to 3.
   // For YOUNG_ANDROID_VERSION 145:
   // - Spelling of "obsfucate" was corrected to obfuscate and BLOCKS_LANGUAGE_VERSION incremented to 20
+  // - FTC_GAMEPAD_COMPONENT_VERSION was incremented to 2.
+  // - FTC_ROBOT_CONTROLLER_COMPONENT_VERSION was incremented to 3.
```

Comment:
@lizlooney Is this safe? Or should bump YaVersion (and reserve it in the master branch)?